planner/core/stats.go

```diff
@@ -862,9 +862,9 @@ func (p *LogicalJoin) getGroupNDVs(colGroups [][]*expression.Column, childStats
 
 // DeriveStats implement LogicalPlan DeriveStats interface.
 // If the type of join is SemiJoin, the selectivity of it will be same as selection's.
-// If the type of join is LeftOuterSemiJoin, it will not add or remove any row. The last column is a boolean value, whose Cardinality should be two.
+// If the type of join is LeftOuterSemiJoin, it will not add or remove any row. The last column is a boolean value, whose ColNDVs should be two.
```

Comment:
```suggestion
// If the type of join is LeftOuterSemiJoin, it will not add or remove any row. The last column is a boolean value, whose NDV should be two.
```